deploy/crds/toolchain_v1alpha1_nstemplateset_crd.yaml

```diff
@@ -45,20 +45,6 @@ spec:
         spec:
           description: NSTemplateSetSpec defines the desired state of NSTemplateSet
           properties:
-            clusterResources:
```

Comment:
Why `clusterResources` are being removed from the CRD?

Reply:
yes, I just noticed that. Not sure why :/